### PR TITLE
IMU component

### PR DESCRIPTION
**Pull request by daniilpastukhov**
Implement IMU component, add an example how it works

### CODE REVIEW

File: depthai_sdk/examples/imu.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from depthai_sdk.classes.packets import IMUPacket
```
Comment (Erol444) [inline depthai_sdk/examples/imu.py:1]: Could IMUPacket be accessible from depthai_sdk? Not sure pros/cons of it

File: depthai_sdk/src/depthai_sdk/classes/packets.py [hunk 2]
```diff
@@ -146,3 +152,30 @@ def _add_detection(self, img_det: dai.ImgDetection, bbox: np.ndarray, txt:str, c
...
+                imu_data.acceleroMeter.y,
+                imu_data.acceleroMeter.z
+            )
+
+            gyroscope_str = 'Gyroscope [rad/s]: (x: %.2f, y: %.2f, z: %.2f)' % (
+                imu_data.gyroscope.x,
+                imu_data.gyroscope.y,
+                imu_data.gyroscope.z
+            )
+            dai.IMUPacket
```
Comment (Erol444) [inline depthai_sdk/src/depthai_sdk/classes/packets.py:178]: What's this?

File: depthai_sdk/src/depthai_sdk/components/imu_component.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        Returns: None
+        """
+        sensors = sensors or [dai.IMUSensor.ACCELEROMETER_RAW, dai.IMUSensor.GYROSCOPE_RAW]
+
+        self.node.enableIMUSensor(sensors=sensors, reportRate=report_rate)
+        self.node.setBatchReportThreshold(batchReportThreshold=batch_report_threshold)
+        self.node.setMaxBatchReports(maxBatchReports=max_batch_reports)
+        self.node.enableFirmwareUpdate(enable_firmware_update)
+
+    def _update_device_info(self, pipeline: dai.Pipeline, device: dai.Device, version: dai.OpenVINO.Version):
```
Comment (Erol444) [inline depthai_sdk/src/depthai_sdk/components/imu_component.py:43]: You can add `# TODO: check whether we have IMU (and which sensors are supported) once API supports it`

**Review by Erol444 — APPROVED** [review]
LGTM:)

File: depthai_sdk/src/depthai_sdk/classes/packets.py (hunk 2 above)
Comment (daniilpastukhov) [reply depthai_sdk/src/depthai_sdk/classes/packets.py]: Fixed, probably pasted by accident

File: depthai_sdk/examples/imu.py (hunk 1 above)
Comment (daniilpastukhov) [reply depthai_sdk/examples/imu.py]: Done

File: depthai_sdk/src/depthai_sdk/components/imu_component.py (hunk 3 above)
Comment (daniilpastukhov) [reply depthai_sdk/src/depthai_sdk/components/imu_component.py:45]: Added to config_imu()

**Pull request merged**